source/includes/steps-schedule-balancer-window.yaml

```diff
@@ -0,0 +1,75 @@
+ref: connect-mongos
+stepnum: 1
+level: 3
+title: "Connect to :program:`mongos` using the :program:`mongo` shell."  
+content: |
+  You can connect to any :program:`mongos` in the cluster.
+---
+ref: use-config
+stepnum: 2
+level: 3
+title: "Switch to the :ref:`config-database`."
+action:
+  - pre: |
+      Issue the following command to switch to the config database.
+    language: sh
+    code: |
+      use config
+---
+ref: ensure-balancer-not-stopped
+stepnum: 3
+level: 3
+title: "Ensure that the balancer is not ``stopped``."
+action:
+  - pre: |
+      The balancer will not activate in the ``stopped`` state.
+      To ensure that the  balancer
+      is not ``stopped``, use :method:`sh.setBalancerState()`,
+      as in the following:
+    language: sh
+    code: |
+      sh.setBalancerState( true )
+    post: |
+      The balancer will not start if you are outside
```

Comment:
I did contemplate making this a `note` and adding more detail to why that's relevant (if you're updating the window, and you try to start it but are outside of the time, it won't start). Alternatively, we could maybe kill the `post` entirely.